docs/explanations/how_to_interpret_the_security_status_command.md

```diff
@@ -1,7 +1,186 @@
 # What does `security-status` do?
 
+## Command output
+
 The `security-status` command is used to get an overview of the packages
-installed on your machine.
+installed on your machine, and an overview of the security coverage for those
+packages.
+
+If you run the `pro security-status` command, the first blocks of information
+you see look like:
+
+```
+2871 packages installed:
+     2337 packages from Ubuntu Main/Restricted repository
+     504 packages from Ubuntu Universe/Multiverse repository
+     8 packages from third parties
+     22 packages no longer available for download
+
+To get more information about the packages, run
+    pro security-status --help
+for a list of available options.
+```
+
+Those are counts for the `apt` packages installed in the system, sorted
+between the packages in main, universe, third party packages, and packages
```

Comment:
I'm still not 100% sure if main/universe/restricted are meant to be capitalised or not (I've seen both), but it would be good if we could choose which we want to go with and use only that (for consistency). Lmk whether we want to use capitalised/decapitalised and I'll highlight any that don't match.

Reply:
/me is sorry - you right I will go all lowercase

Reply:
but now I see the command output has it uppercase - and it was like that since, well, the old commands were there :/